src/datatypes/Draft.ts

```diff
@@ -26,80 +26,6 @@ export type DraftState = {
   steps: DraftStep[];
 }[][];
 
-export const DEFAULT_STEPS: DraftStep[] = [
```

Comment:
Happy to adjust, but it felt odd to me to have logic in the datatypes file so I moved them to draftutils and added tests.